### PR TITLE
Update hydra, tiamat, dhcp/dns, add TalDos, update IP allocations

### PR DESCRIPTION
**Pull request by lavajuno**
- Added and updated information on pages for hydra and tiamat
- Added page for TalDos
- Updated DHCP & DNS pages to point to TalDos
- Added DHCP configuration
- Updated IP allocations to reflect Talos and TalDos's new IPs

### CODE REVIEW

**Comment by Alextopher** [comment]
You can add this to the _typos.toml file to remove the error on the mirror page. I never had a chance to share this.

```
[default]
extend-ignore-identifiers-re = [
    # Ignore mirror hard disk names
    "2RW103_ZL2*",
]
```

**Comment by lavajuno** [comment]
Just put your addition in _typos.toml. Thank you!

**Review by carykees98 — APPROVED** [review]

File: src/services/dhcp.md [hunk 1]
```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
...
+#  option domain-name-servers ns1.internal.example.org;
+#  option domain-name "internal.example.org";
+#  option routers 10.5.5.1;
+#  option broadcast-address 10.5.5.31;
+#  default-lease-time 600;
+#  max-lease-time 7200;
+#}
+
+# Hosts which require special configuration options can be listed in
+# host statements.   If no address is specified, the address will be
```
Comment (carykees98) [inline src/services/dhcp.md:83]:
```suggestion
# host statements. If no address is specified, the address will be
```

File: src/services/dhcp.md [hunk 2]
```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
...
+#host passacaglia {
+#  hardware ethernet 0:0:c0:5d:bd:95;
+#  filename "vmunix.passacaglia";
+#  server-name "toccata.example.com";
+#}
+
+# Fixed IP addresses can also be specified for hosts.   These addresses
+# should not also be listed as being available for dynamic assignment.
+# Hosts for which fixed IP addresses have been specified can boot using
+# BOOTP or DHCP.   Hosts for which no fixed address is specified can only
```
Comment (carykees98) [inline src/services/dhcp.md:96]:
```suggestion
# BOOTP or DHCP. Hosts for which no fixed address is specified can only
```

File: src/services/dhcp.md [hunk 3]
```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
...
+# be booted with DHCP, unless there is an address range on the subnet
+# to which a BOOTP client is connected which has the dynamic-bootp flag
+# set.
+#host fantasia {
+#  hardware ethernet 08:00:07:26:c0:a5;
+#  fixed-address fantasia.example.com;
+#}
+
+# You can declare a class of clients and then do address allocation
+# based on that.   The example below shows a case where all clients
```
Comment (carykees98) [inline src/services/dhcp.md:106]:
```suggestion
# based on that. The example below shows a case where all clients
```

**Review by sophiacarlone — APPROVED** [review]

File: src/services/dhcp.md [hunk 4]
```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
...
+authoritative;
+
+# Use this to send dhcp log messages to a different log file (you also
+# have to hack syslog.conf to complete the redirection).
+#log-facility local7;
+
+# No service will be given on this subnet, but declaring it helps the 
+# DHCP server to understand the network topology.
+
+#subnet 10.152.187.0 netmask 255.255.255.0 {
```
Comment (carykees98) [inline src/services/dhcp.md:51]:
```suggestion
# subnet 10.152.187.0 netmask 255.255.255.0 {
```

File: src/services/dhcp.md [hunk 5]
```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
...
+#}
```
Comment (carykees98) [inline src/services/dhcp.md:52]:
```suggestion
# }
```

File: src/services/dhcp.md [hunk 6]
```diff
@@ -20,6 +23,109 @@ The Internet Systems Consortium's implementation of a DHCP server is good enough
...
+# have support for DDNS.)
+ddns-update-style none;
+
+# If this DHCP server is the official DHCP server for the local
+# network, the authoritative directive should be uncommented.
+authoritative;
+
+# Use this to send dhcp log messages to a different log file (you also
+# have to hack syslog.conf to complete the redirection).
+#log-facility local7;
```
Comment (carykees98) [inline src/services/dhcp.md:46]:
```suggestion
# log-facility local7;
```

**Review by carykees98 — CHANGES_REQUESTED** [review]

File: src/services/dhcp.md (hunk 5 above)
Comment (carykees98) [reply src/services/dhcp.md:52]: not sure if the space should be there or not but wanted to make sure before the merge

File: src/services/dhcp.md (hunk 4 above)
Comment (carykees98) [reply src/services/dhcp.md:51]: not sure if the space should be there or not but wanted to make sure before the merge

File: src/services/dhcp.md (hunk 6 above)
Comment (carykees98) [reply src/services/dhcp.md:46]: not sure if the space should be there or not but wanted to make sure before the merge

**Comment by Alextopher** [comment]
I wonder if you could shrink the DHCP config removing the comments. I sometimes use COSI resources as references for how to do system administration tasks, and the DHCP section is 75% of the way to being a tutorial 😃

**Comment by lavajuno** [comment]
@carykees98 Committed your changes to dhcp.md

**Comment by lavajuno** [comment]
> I wonder if you could shrink the DHCP config removing the comments. I sometimes use COSI resources as references for how to do system administration tasks, and the DHCP section is 75% of the way to being a tutorial 😃

Definitely a good idea, I just pasted the contents of the file in. I'm definitely not opposed to making book entries more tutorial-like in the future, maybe that will be my project for break.

**Pull request merged**